### PR TITLE
backward_ros: 1.0.2-1 in 'rolling/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by saikishor**
Increasing version of package(s) in repository backward_ros to 1.0.2-1:

- upstream repository: https://github.com/pal-robotics/backward_ros.git
- release repository: https://github.com/pal-gbp/backward_ros-release.git
- distro file: rolling/distribution.yaml
- bloom version: 0.10.7
- previous version for package: 1.0.1-2

    
# backward_ros
    Fix name mismatch warnings
    Contributors: Jordan Palacios, Vatan Aksoy Tezer

### CODE REVIEW

File: rolling/distribution.yaml [hunk 1]
```diff
@@ -385,8 +385,8 @@ repositories:
     release:
       tags:
         release: release/rolling/{package}/{version}
-      url: https://github.com/ros2-gbp/backward_ros-release.git
-      version: 1.0.1-2
+      url: https://github.com/pal-gbp/backward_ros-release.git
```
Comment (hidmic) [inline rolling/distribution.yaml:388]: @saikishor IIRC our Rolling distribution infrastructure requires release repos to be under the `ros2-gbp` organization. CC @nuclearsandwich.

Comment (nuclearsandwich) [reply rolling/distribution.yaml:388]: @saikishor the current access is described by the configuration here: https://github.com/ros2-gbp/ros2-gbp-github-org/blob/latest/pal_robotics.tf 

You can use the update release team issue template when [creating an issue](https://github.com/ros2-gbp/ros2-gbp-github-org/issues/new/choose) on that project to request changes to the team composition.

Comment (saikishor) [reply rolling/distribution.yaml:388]: @nuclearsandwich Thank you for the info, will request for access in an issue.

Thank you

Comment (saikishor) [reply rolling/distribution.yaml:388]: Requested for access : https://github.com/ros2-gbp/ros2-gbp-github-org/issues/35

Once we are given access, I will do the release again and update this PR with new changes. Thank you.

Comment (saikishor) [reply rolling/distribution.yaml]: @hidmic and @nuclearsandwich, I have made the release in the `ros2-gbp`  organization repository.

**Comment by hidmic** [comment]
@saikishor mind to update repository links in this PR's description to match? As a rule of thumb, I'd say it's preferable to let `bloom` open another PR instead of tinkering with it too much (and risking inconsistency).

**Comment by saikishor** [comment]
> @saikishor mind to update repository links in this PR's description to match? As a rule of thumb, I'd say it's preferable to let `bloom` open another PR instead of tinkering with it too much (and risking inconsistency).

@hidmic I can understand, I will make a new release with bloom. Thank you

**Comment by saikishor** [comment]
@hidmic I have opened a new MR : https://github.com/ros/rosdistro/pull/32647 that is automatically generated by the bloom.
I will be closing this MR.

**Pull request closed without merging**